lib/app/views/partials/_ribbon.html.erb

```diff
@@ -1,6 +1,6 @@
 <% config = Ribbonit.configuration %>
 
-<% if Rails.env.development? || Rails.env.send("#{config.staging_name}?") %>
+<% if config.themes.keys.include? Rails.env %>
```

Comment:
Your change doesn't work as you are comparing a `Symbol` with a `String`...
Moreover, there are another unexpected bugs I am still investigating :)